micro-rdk-installer/src/main.rs

```diff
@@ -349,18 +351,29 @@ fn main() -> Result<(), Error> {
             )?;
         }
         Some(Commands::WriteFlash(args)) => {
-            let config = Config::load().map_err(|err| Error::SerialConfigError(err.to_string()))?;
             let tmp_path = tempfile::NamedTempFile::new()
                 .map_err(Error::FileError)?
                 .path()
                 .to_path_buf();
             let app_path = match &args.binary_path {
                 Some(path) => PathBuf::from(path),
                 None => {
+                    if args.version.is_none() {
+                        let use_latest = dialoguer::Confirm::new()
+                            .with_prompt("Download and flash the latest Micro-RDK release?")
```

Comment:
What happens here if stdin is not a tty?

Reply:
confirmed with `micro-rdk-installer write-flash < /dev/null` that it doesn't hang, but somehow still results in an interactive prompt.

Reply:
Alright, not worth spending more time on that I can see.